qa/pull-tester/rpc-tests.py

```diff
@@ -118,6 +119,7 @@
     'p2p-versionbits-warning.py',
 ]
 testScriptsExt = [
+    'bip9-softforks.py',
```

Comment:
IMO: bip9 itself is not a softfork and could therefore be in the "normal" `testScripts` array.